lldb/tools/lldb-dap/SourceBreakpoint.h

```diff
@@ -52,14 +37,33 @@ struct SourceBreakpoint : public Breakpoint {
   static bool BreakpointHitCallback(void *baton, lldb::SBProcess &process,
                                     lldb::SBThread &thread,
                                     lldb::SBBreakpointLocation &location);
```

Comment:
Does `BreakpointHitCallback` need to be in the header here? I guess that may be a functional change if we move this into the .cpp file. 

I also think other breakpoint types can also have log callbacks, but I'm not sure if they're all handled consistently.